app/src/lib/editors/win32.ts

```diff
@@ -582,41 +541,31 @@ async function findApplication(editor: WindowsExternalEditor) {
     }
   }
 
-  return findJetBrainsToolboxApplication(editor)
+  return undefined
 }
 
-/**
- * Find JetBrain products installed through JetBrains Toolbox
- */
-async function findJetBrainsToolboxApplication(editor: WindowsExternalEditor) {
-  if (!editor.jetBrainsToolboxScriptName) {
-    return null
-  }
-
-  const toolboxRegistryReference = [
-    CurrentUserUninstallKey('toolbox'),
-    Wow64LocalMachineUninstallKey('toolbox'),
-  ]
-
-  for (const { key, subKey } of toolboxRegistryReference) {
-    const keys = enumerateValues(key, subKey)
-    if (keys.length > 0) {
-      const editorPathInToolbox = Path.join(
-        getKeyOrEmpty(keys, 'UninstallString'),
-        '..',
-        '..',
-        'scripts',
-        `${editor.jetBrainsToolboxScriptName}.cmd`
-      )
-      const exists = await pathExists(editorPathInToolbox)
-      if (exists) {
-        return editorPathInToolbox
-      }
+const getJetBrainsToolboxEditors = memoizeOne(async () => {
+  const re = /^JetBrains Toolbox \((.*)\)/
+  const editors = new Array<WindowsExternalEditor>()
+
+  for (const key of enumerateKeys(HKEY.HKEY_CURRENT_USER, uninstallSubKey)) {
```

Comment:
Should we also check `wow64UninstallSubKey` like in the previous implementation?